numpy/_build_utils/tempita/_tempita.py

```diff
@@ -293,8 +293,7 @@ def _interpret_for(self, vars, expr, content, ns, out, defs):
             else:
                 if len(vars) != len(item):
                     raise ValueError(
-                        "Need %i items to unpack (got %i items)"
-                        % (len(vars), len(item))
+                        f"Need {len(vars)} items to unpack (got {len(item)} items)"
```

Comment:
No need to touch this, it is a vendored tool that we use internally for building the random module's templated code. Maybe someday we can use the new t-string formatting.